articles/overview/what-is-quantum-computing.md

```diff
@@ -0,0 +1,46 @@
+---
+title: What is quantum computing?
+description: 
+author: natke
+ms.author:  
+ms.date: 10/14/2019
+ms.topic: article
+uid: microsoft.quantum.overview.whatis
+---
+
+# What is quantum computing?
+
+Quantum computing is the application of quantum physics to the field of computer science. Quantum computing might sound daunting, but with the right resources you can start building quantum applications today.
+
+## The qubit
+
+The basic unit of quantum computing is the `qubit`. Unlike a `bit` in classical computing, which can be `0` or `1`, a qubit can be in an infinite number of states - a **superposition** of both 0 and 1.
```

Comment:
Quick question, but do we want to use code formatting for "qubit" and "bit"? With respect to the technical content, while it is definitely true and correct that a qubit can be in an infinite number of states, I've generally found that fact can be fairly easily misunderstood such that I would suggest either more qualifications to the statement here or deferring that to the quantum concepts guide.

Reply:
Good question, we should decide on a standard here. The fixed width formatting is used for more than code, often used to denote terminology (and if a repo is localized then these terms do not get translated)